package-lock.json

```diff
@@ -0,0 +1,1318 @@
+{
```

Comment:
What is package-lock.json for?

Reply:
I don't know exactly. It first appeared i believe with Node 8 (npm XXX). Npm outputs a line which recommends you put it under version control.